filt_num: [0.00000000000000000000e+00, 5.39779723216088247685e-15, 5.93515488195678079867e-14, 5.93273383696824827531e-14, 5.39119443032787735186e-15]
filt_den: [1.00000000000000000000e+00, -3.99796032041345084096e+00, 5.99388272094646001875e+00, -3.99388447991865680464e+00, 9.97962079385777300899e-01]
Dfilt_num: [0.00000000000000000000e+00, 2.15889865929285457672e-11, 6.47229286125692044445e-11, -6.47449381520994586328e-11, -2.15669770533982237243e-11]
Dfilt_den: [1.00000000000000000000e+00, -3.99796032041345084096e+00, 5.99388272094646001875e+00, -3.99388447991865680464e+00, 9.97962079385777300899e-01]
st: 1.000000e-03